.github/workflows/docker-unified.yml

```diff
@@ -470,11 +470,11 @@ jobs:
       - name: Pull images in background
         run: |
           docker pull acryldata/datahub-actions:head &
-          docker pull confluentinc/cp-kafka:7.2.0 &
+          docker pull confluentinc/cp-kafka:7.2.2 &
```

Comment:
just bringing in line with current quickstart. Will think of a better way so we don't have to do this manually.

Reply:
This step is not required, my guess is that it is to eagerly pull them but they would be pulled as needed later in any case. Not sure how much time this saves. Alternative would be to use the docker compose and the docker compose command to fetch only here `docker compose pull <yamls>`

Reply:
without this the quickstart was taking 6.5 mins. After this was added it took 4.5 minutes.

Reply:
checking again it seems the quickstart time had gone down between my tests. So seems like this isn't required. Will remove this pre-fetch.